persistence/src/androidMain/kotlin/com/wire/kalium/persistence/db/PlatformDatabaseData.kt

```diff
@@ -27,3 +33,32 @@ import android.content.Context
 actual class PlatformDatabaseData(
     val context: Context
 )
+
+@Suppress("LongParameterList")
+fun databaseDriver(
+    context: Context,
+    dbName: String,
+    isEncryptionEnabled: Boolean,
+    passphrase: ByteArray? = null,
+    schema: SqlSchema<QueryResult.Value<Unit>>,
+    config: DriverConfigurationBuilder.() -> Unit = {}
+): SqlDriver {
+    val driverConfiguration = DriverConfigurationBuilder().apply(config)
+    val enableWAL = driverConfiguration.isWALEnabled
+    return if (isEncryptionEnabled) {
```

Comment:
`isEncryptionEnabled` can be removed 😄 

```suggestion
    passphrase: ByteArray? = null,
    schema: SqlSchema<QueryResult.Value<Unit>>,
    config: DriverConfigurationBuilder.() -> Unit = {}
): SqlDriver {
    val driverConfiguration = DriverConfigurationBuilder().apply(config)
    val enableWAL = driverConfiguration.isWALEnabled
    return if (passphrase != null) {
```

Reply:
Good idea, just needed to lift the check for android, depending on kalium configs.